dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -4,12 +4,6 @@ This benchmarking script will allow you to run and benchmark the performance of
 
 ## Getting started
 
-### Installation
-
-```shell
-pip install gcs-torch-dataflux gcsfs
```

Comment:
It looks like there are a few lingering references to `gcs-torch-dataflux`, let's scrub the rest of them in this PR.

https://github.com/search?q=repo%3AGoogleCloudPlatform%2Fdataflux-pytorch%20gcs-torch-dataflux&type=code

Reply:
The others look mostly right for now. The checkpointing single-node benchmark probably not, but that readme doesn't have the `pip install .` and `pip install -r ...` commands listed that this one does, so it may be needed there.

The key difference is that running `pip install gcs-torch-dataflux` is going to install the public version from PyPi and run against it, vs. the way this one is set up will use your local changes.